plugins/uberdev/commands/review-pr.md

```
@@ -612,13 +612,13 @@ On any artifact-emission failure (anchor commit fails — pre-commit hook reject
 RUN_ID="$(date +%Y%m%d-%H%M%S)-$(git rev-parse --short HEAD)"
 ```
 
-This mirrors the convention in `skills/merge/SKILL.md:209` (Phase 3.3ii scratch worktree path). Before any path concatenation, validate `<run-id>` against the regex:
+This mirrors the convention in `skills/merge-pipeline/SKILL.md:209` (Phase 3.3ii scratch worktree path). Before any path concatenation, validate `<run-id>` against the regex:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)